### PR TITLE
Initial oscilloscope UI tweaks

### PR DESCRIPTION
**Pull request by mkruselj**
Biggified the whole thing
Various text label tweaks, some parameter range and value display tweaks
Retrigger Threshold didn't gray out when Free/Internal modes were selected
Use MSEG Panel skin color for parameter area background
Larger font used for sliders, like in the rest of the GUI

### CODE REVIEW

File: src/surge-xt/gui/overlays/Oscilloscope.cpp [hunk 1]
```diff
@@ -193,6 +206,7 @@ void WaveformDisplay::process(std::vector<float> data)
     int triggerLimit =
         static_cast<int>(std::pow(10.f, params_.trigger_limit * 4.f)); // 0=>1, 1=>10000
     float triggerSpeed = std::pow(10.f, 2.5f * params_.trigger_speed - 5.f);
+    printf("%.5f\n", triggerSpeed);
```
Comment (mx) [inline src/surge-xt/gui/overlays/Oscilloscope.cpp:209]: Looks like a debug printf, should remove this.

File: src/surge-xt/gui/overlays/Oscilloscope.cpp [hunk 2]
```diff
@@ -467,21 +490,27 @@ void SpectrumDisplay::resized()
...
 
 void SpectrumDisplay::updateScopeData(internal::FftScopeType::iterator begin,
                                       internal::FftScopeType::iterator end)
 {
     // Data comes in as gain.
     std::lock_guard l(data_lock_);
+
     // Decay existing data, and move new data in if it's larger.
-    const float decay = (1.f - params_.decay_rate);
+    const float decay = (1.f - (params_.decay_rate * 0.9f));
```
Comment (mx) [inline src/surge-xt/gui/overlays/Oscilloscope.cpp:503]: With this change we no longer can get the instant-decay behavior when decay_rate = 1 or maxes behavior when decay_rate = 0.

File: src/surge-xt/gui/overlays/Oscilloscope.cpp [hunk 3]
```diff
@@ -1260,19 +1359,23 @@ void Oscilloscope::Background::paintWaveformBackground(juce::Graphics &g)
         std::string minus = "-";
         std::string plus = "+";
         std::stringstream gain;
-        gain << std::fixed << std::setprecision(2) << 1.f / waveform_params_.gain();
+        const unsigned prec = waveform_params_.gain() > 1.f ? 2 : 0;
```
Comment (mx) [inline src/surge-xt/gui/overlays/Oscilloscope.cpp:1362]: I'd explicitly label this as an unsigned int. I think raw unsigned is a deprecated C-ism (here and below).

File: src/surge-xt/gui/overlays/Oscilloscope.cpp (hunk 2 above)
Comment (mkruselj) [reply src/surge-xt/gui/overlays/Oscilloscope.cpp:503]: OK I will then just adjust the scaling to be sqrt of params_.decay_rate.

**Pull request merged**